cucumber/features/careerPage.feature

```diff
@@ -2,3 +2,30 @@ Feature: EPAM job searching
   As a Job searcher
   I want to browser through EPAM Job offers by various criteria
   So I can find to best fitting offer for me
+
+Scenario Outline: Search for <PositionName> in <City>
+  Given the career page is opened
+  Then the cookie bar should be closed
```

Comment:
This step should be a "When" step, it is an action step. Could you please modify it?

Reply:
done